servicedirectory/create_endpoint.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicedirectory
+
+// [START servicedirectory_create_endpoint]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	servicedirectory "cloud.google.com/go/servicedirectory/apiv1beta1"
+	sdpb "google.golang.org/genproto/googleapis/cloud/servicedirectory/v1beta1"
+)
+
+func createEndpoint(w io.Writer, projectId string) error {
+	location := "us-east4"
+	namespaceId := "golang-test-namespace"
+	serviceId := "golang-test-service"
+	endpointId := "golang-test-endpoint"
+
+	ctx := context.Background()
+	// Create a registration client.
+	client, err := servicedirectory.NewRegistrationClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Create an Endpoint.
+	createEndpointReq := &sdpb.CreateEndpointRequest{
+		Parent:     fmt.Sprintf("projects/%s/locations/%s/namespaces/%s/services/%s", projectId, location, namespaceId, serviceId),
+		EndpointId: endpointId,
+		Endpoint: &sdpb.Endpoint{
+			Address: "10.10.10.10",
```

Comment:
Confirming this address is OK to use here. Should we use 8.8.8.8 (Google DNS)?

Reply:
switched to 8.8.8.8